### DISCUSSION TITLE
Welcome to pairtree Discussions!

### DISCUSSION THREAD

**Discussion by jwintersinger** [morrislab/pairtree#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LayalYasin** [comment]
Hi !
what is the minimum number of cancer samples required from one patient to be able to use the Pairtree algorithm ? is it 3 samples ?

**Reply by ethanumn** [reply]
Hi there -- 

One cancer sample per patient is the minimum number of samples required to use Pairtree.

In practice, Pairtree can handle up to around 100 samples per patient!

**Reply by LayalYasin** [reply]
thanks for your answer ! and is there any minimum-required coverage for the somatic mutations ?
